docs/topics/permissions.rst

```diff
@@ -2,15 +2,18 @@
 Permissions
 ###########
 
-In django CMS you can set two types of permissions:
+In django CMS you can set three types of permissions:
 
 1. View restrictions for restricting front-end view access to users
-2. Page permissions for allowing staff users to only have rights on certain sections of certain sites
+1. Page permissions for allowing staff users to only have rights on certain sections of certain sites
+1. Mode permission for allowing staff users to only edit existing content and not add new one
```

Comment:
Consider: "Mode permission which when left unset, restricts staff users to only editing, not adding new content" to replace the above line.